homework/index.js

```diff
@@ -29,19 +29,149 @@
     });
     return elem;
   }
+  // ==========start===========
+  /* cSpell: disable */
```

Comment:
Please enable **Spell Checker** and fix spelling errors if there is any.

Reply:
I did it at the end of the code, I used naming which are considered as spell error. I will avoid that next time.